server/backend/api/app/routers/router_idim_proxy.py

```diff
@@ -1,33 +1,44 @@
 import logging
 
 from api.app.integration.idim_proxy import IdimProxyService
-from api.app.routers.router_guards import (get_current_requester,
-                                           internal_only_action)
-from api.app.schemas import IdimProxyIdirInfo, IdimProxySearchParamIdir
+from api.app.routers.router_guards import get_current_requester, internal_only_action
+from api.app.schemas import IdimProxyIdirInfo, IdimProxySearchParam, IdimProxyBceidInfo
 from fastapi import APIRouter, Depends, Query
 
-ERROR_EXTERNAL_USER_ACTION_PROHIBITED = "external_user_action_prohibited"
 
 LOGGER = logging.getLogger(__name__)
 
 router = APIRouter()
 
-@router.get("/idir", response_model=IdimProxyIdirInfo, dependencies=[Depends(internal_only_action)])
+
+@router.get(
+    "/idir",
+    response_model=IdimProxyIdirInfo,
+    dependencies=[Depends(internal_only_action)],
```

Comment:
What does this internal_only_action do?

Reply:
this is an exiting check, only IDIR user can search for IDIR